core/src/main/java/nl/nn/adapterframework/receivers/JavaListener.java

```diff
@@ -47,8 +47,8 @@
 
 // TODO: When anchors are supported by the Frank!Doc, link to https://github.com/ibissource/ibis-servicedispatcher
 /**
- * Use this listener to receive messages from other adapters or a scheduler within the same Frank! Application or from other components residing in the same JVM.
- * JavaListeners can receive calls made via de ibis-servicedispatcher, which should be located on the JVM classpath to receive calls from other components in the JVM. If you want to call an adapter in the same Frank! Application, consider using the IbisLocalSender.
+ * Use this listener to receive messages from other adapters or a scheduler within the same Frank or from other components residing in the same JVM.
```

Comment:
Ik denk niet dat deze wijziging voor meer begrip of minder verwarring zal zorgen.

Reply:
Aan de telefoon bespraken we dat het beste `Frank! Application` vervangen kon worden door `Frank-application`

Reply:
Gedaan.